### PR TITLE
devtools の LYRA サポートを削除する

### PR DESCRIPTION
**Pull request by tnamao**
sora-js-sdk の対応するバージョンがリリースされて (package.json の依存を更新して) からマージ可能です

### 変更履歴

- [CHANGE] オーディオコーデック `LYRA` の設定を削除する
  - 関連するコードと `service-worker.js` の削除
  - next.config.js から不要な設定の削除
  - @tnamao

### CODE REVIEW

**Comment by sile** [comment]
copilot summary

---

This pull request involves significant changes to the codebase, primarily focused on removing the LYRA audio codec settings. The changes include the deletion of the LYRA codec settings from various files, removal of related code and files, and the updating of documentation to reflect these changes.

Removal of LYRA audio codec settings:
* [`CHANGES.md`](diffhunk://#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R14-R17): Removed the LYRA audio codec settings and associated code, including `service-worker.js`. Also removed unnecessary settings from `next.config.js`.
* [`DOCUMENT.md`](diffhunk://#diff-3eeac631f059f6cefe88d6d609987fc4b3e0da69005c77443d0ec3541d6efd63L123-L126): Removed the `audioLyraParamsBitrate` section which dealt with the LYRA codec bitrate settings.
* [`instructions.json`](diffhunk://#diff-d4a451ad2863e5577b7cc521ab817f447f4a7c53c11bf6281e7fc34511f35f2aL86-L88): Removed the `audioLyraParamsBitrate` section from the instructions.

Deletion of related code and files:
* [`public/service-worker.js`](diffhunk://#diff-c7fe7cbf34906a3e945332fd1e6daf8b29668e3d5f56783bb5232647b48480a7L1-L32): Entire file, which was associated with LYRA codec, was deleted.
* [`src/app/actions.ts`](diffhunk://#diff-f614e3891e365664ae453d152ad62d608f4f5fa9708ead03b11241130e00af00L1916-L1945): Removed LYRA codec initialization and related actions. [[1]](diffhunk://#diff-f614e3891e365664ae453d152ad62d608f4f5fa9708ead03b11241130e00af00L1916-L1945) [[2]](diffhunk://#diff-f614e3891e365664ae453d152ad62d608f4f5fa9708ead03b11241130e00af00L1993)
* [`src/components/DevtoolsPane/AudioLyraParamsBitrateForm.tsx`](diffhunk://#diff-494221da73b57d13ecccb87b8a6630f076139128a9ee7f6e8dcc079dbd968fa5L1-L40): Entire file, which was associated with LYRA codec, was deleted.

Updates to documentation and other files:
* [`src/app/slice.ts`](diffhunk://#diff-45bfbe03821b84e4fcd3d64b38e126de5d7b7a7a7537adf8a7088fb9c9c6194eL67): Removed the `audioLyraParamsBitrate` from the initial state and the slice. [[1]](diffhunk://#diff-45bfbe03821b84e4fcd3d64b38e126de5d7b7a7a7537adf8a7088fb9c9c6194eL67) [[2]](diffhunk://#diff-45bfbe03821b84e4fcd3d64b38e126de5d7b7a7a7537adf8a7088fb9c9c6194eL621-L626)
* [`src/constants.ts`](diffhunk://#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199L9-R9): Removed `LYRA` from the `AUDIO_CODEC_TYPES` and deleted `AUDIO_LYRA_PARAMS_BITRATES`. [[1]](diffhunk://#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199L9-R9) [[2]](diffhunk://#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199L124-L125)
* [`src/types.ts`](diffhunk://#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aL87): Removed `audioLyraParamsBitrate` from the `SoraDevtoolsState` and `ConnectionOptionsState` types. [[1]](diffhunk://#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aL87) [[2]](diffhunk://#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aL355)
* [`src/utils.ts`](diffhunk://#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351L197-L200): Removed the parsing and creation of `audioLyraParamsBitrate` from the query string and connect options respectively. [[1]](diffhunk://#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351L197-L200) [[2]](diffhunk://#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351L759-L765)

File: src/app/actions.ts [hunk 1]
```diff
@@ -1913,36 +1905,6 @@ const stopLocalAudioTrack = (
...
-    // 実際に Lyra コーデックの音声を送信・受信するまでは特別な処理は発生しない。
-    // 未対応環境だった場合には、Lyra コーデックが必要になった段階でエラーが発生する。
-    const wasmPath = 'https://lyra-wasm.shiguredo.app/2022.2.0/'
-    const modelPath = wasmPath
-    Sora.initLyra({ wasmPath, modelPath })
-
-    // lyra-wasm は SharedArrayBuffer を使っているので、それを有効にするために必要な
-    // HTTP 応答ヘッダの設定を行うサービスワーカを登録する。
-    if ('serviceWorker' in navigator) {
-      navigator.serviceWorker.register('./service-worker.js').then((registration) => {
```
Comment (sile) [inline src/app/actions.ts:1930]: 実害はないと思うので、単なる自分用のメモコメントですが、このコードを消してもすでに登録済みのサービスワーカーは（ブラウザのキャッシュを削除するまで）残り続けそうですね。

**Review by sile — APPROVED** [review]
良さそうに見えました

**Comment by tnamao** [comment]
sora-js-sdk を 2024.1.0-canary.0 の Lyra 削除バージョンに入れ替えました
マージ可能になったのでマージします

**Pull request merged**